pkg/controller/replication/replication_controller.go

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 // ### ATTENTION ###
 //
-// ReplicationManager is now just a wrapper around ReplicaSetController,
+// Manager is now just a wrapper around ReplicaSetController,
```

Comment:
I'm not a big fan of blindly following golint errors, I'd prefer we don't change these.